examples/jsm/renderers/common/Background.js

```diff
@@ -2,6 +2,7 @@ import DataMap from './DataMap.js';
 import Color4 from './Color4.js';
 import { Mesh, SphereGeometry, BackSide } from 'three';
 import { vec4, context, normalWorld, backgroundBlurriness, backgroundIntensity, NodeMaterial, modelViewProjection } from '../../nodes/Nodes.js';
```

Comment:
Maybe we could add `maxMipLevel` import here?

Reply:
Right, it should be imported here